src/index.js

```diff
@@ -29,9 +29,19 @@ const overriddenCmps = {
   // 'LiteratureCover.layout': CustomCover,
 };
 
+const externalConfig = {
+  invenioConfig: {
+    authors: {
+      maxDisplay: 42,
+    },
+  },
+  uiConfig: {},
+  searchConfig: {},
+};
+
 ReactDOM.render(
   <OverridableContext.Provider value={overriddenCmps}>
-    <InvenioILSApp />
+    <InvenioILSApp externalConfig={externalConfig} />
```

Comment:
nit: it could be named just `config` or `configuration`, no need for external IMO